src/node/coinstats.cpp

```diff
@@ -41,6 +41,17 @@ static void ApplyStats(CCoinsStats &stats, CHashWriter& ss, const uint256& hash,
     ss << VARINT(0u);
 }
 
+static void ApplyStats(CCoinsStats &stats, nullptr_t, const uint256& hash, const std::map<uint32_t, Coin>& outputs)
```

Comment:
nit ea41a5bdcc49566be0c23ef9362781e70f557bad

```suggestion
static void ApplyStats(CCoinsStats& stats, nullptr_t, const uint256& hash, const std::map<uint32_t, Coin>& outputs)
```

Reply:
done